autoscheduler/scraper/management/commands/scrape_courses.py

```diff
@@ -4,18 +4,89 @@
 from django.core.management import base
 from scraper.banner_requests import BannerRequests
 from scraper.models import Course, Instructor, Section, Meeting
+from course import generate_course_id
 
-def parse_section(section):
-    " parse_course() must be done first, and pass the data for each section here "
+def parse_section(course):
+    #gets values to create object
+    section_id = ''
+    section_subject = course['subject']
+    course_number = course['courseNumber']
+    section_number = ''
+    section_term_code = course['term']
+    
+    section_min_credits = course['creditHourLow']
+    section_max_credits = course['creditHourHigh']
 
-def parse_meeting(meeting):
-    " parse_section() must be done first, and pass the data for each meeting here"
+    section_max_enrollment = course['maximumEnrollment']
+    section_current_enrollment = course['enrollment']
+    section_instructor = ''
+    for i in course['faculty']:
+        instructor_name = i['displayName']
+        section_instructor.join(instructor_name)
 
-def parse_instructor(instructor):
-    " parse_course() must be done first, and pass the data for the instructor here "
+    #creates and saves section object
+    s = Section(id=section_id, subject=section_subject, course_num=course_number, section_num=section_number, term_code=section_term_code, min_credits=section_min_credits, max_credits=section_max_credits, max_enrollment=section_max_enrollment, current_enrollment=section_current_enrollment, instructor=section_instructor)
+    s.save()
+
+    #calls parse meeting for each meeting time in the section
+    for i in course['meetingsFaculty']:
+        parse_meeting(i)
+
+def parse_meeting(i):
+    #gets values to create object
+    meeting_id = ''
+    meeting_crn = i['meetingTime']['courseReferenceNumber']
+
+    meeting_building = i['meetingTime']['building']
```

Comment:
Should we add room number as a field to the meeting model? We have the data.

Reply:
For now I think it's fine if we don't as I don't think it would affect anyone (plus for minimizing distance between classes all we need is the building). We can always add it later if we find a need for it.